src/functionalTest/java/uk/gov/hmcts/reform/feature/BaseTest.java

```diff
@@ -52,8 +49,7 @@ protected RequestSpecification requestSpecification() {
             .auth().preemptive().basic(testAdminUser, testAdminPassword)
             .relaxedHTTPSValidation()
             .baseUri(this.testUrl)
-            .header(HttpHeaders.CONTENT_TYPE, MediaType.APPLICATION_JSON_VALUE)
-            .header(SyntheticHeaders.SYNTHETIC_TEST_SOURCE, SYNTHETIC_SOURCE_HEADER_VALUE);
```

Comment:
we can still use it i suppose - just a way for app insights to know it's a fake request